editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -2908,10 +2918,10 @@ void CanvasItemEditor::_draw_grid() {
 
 				if (last_cell != cell) {
 					Color grid_color;
-					if (primary_grid_steps == 0) {
+					if (primary_grid_step_x == 0) {
```

Comment:
```suggestion
					if (primary_grid_step_x <= 1) {
```
The behavior is the same for 0 and 1, so a general check is better here.